themes/basic-bold/fundamentals/colors.scss

```diff
@@ -1,6 +1,4 @@
 /* Theme colors */
 
-:root {
-  --basic-bold-accent-color-1: #c4d8c4; /* Green */
-  --basic-bold-accent-color-2: #ffea9e; /* Yellow */
-}
+$basic-bold-accent-color-1: #c4d8c4; /* Green */
+$basic-bold-accent-color-2: #ffea9e; /* Yellow */
```

Comment:
These variables are not used anymore.

Reply:
Thanks for pointing that out. I will remove them.